spec/mutations/sequences/create_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'spec_helper'
+
+describe Steps::Create do
+  let(:user) { FactoryGirl.create(:user) }
+  let(:mutation) { Sequences::Create }
+  let(:step) do
+    { message_type: 'move_rel',
+      command: { action: 'MOVE RELATIVE',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.